<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MediaElch</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../MediaElch.qrc">
    <normaloff>:/img/MediaElch.png</normaloff>:/img/MediaElch.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget {
	background-image: url(:/img/pattern2.png);
}

#menuWidget {
	background-color: rgb(51, 57, 61);
	border-right: 1px solid rgb(34, 34, 34);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="menuWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="buttonMovies">
         <property name="toolTip">
          <string>Movies</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border: 0;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 0px
}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../MediaElch.qrc">
           <normaloff>:/img/video_menuActive.png</normaloff>:/img/video_menuActive.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>52</width>
           <height>52</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonMovieSets">
         <property name="toolTip">
          <string>Movie Sets</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border: 0;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 0px
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../MediaElch.qrc">
           <normaloff>:/img/movieSets_menu.png</normaloff>:/img/movieSets_menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>52</width>
           <height>52</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonGenres">
         <property name="toolTip">
          <string>Genres</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border: 0;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 0px
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../MediaElch.qrc">
           <normaloff>:/img/genre_menu.png</normaloff>:/img/genre_menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>52</width>
           <height>52</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonCertifications">
         <property name="toolTip">
          <string>Certifications</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border: 0;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 0px
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../MediaElch.qrc">
           <normaloff>:/img/certification2_menu.png</normaloff>:/img/certification2_menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>52</width>
           <height>52</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonTvshows">
         <property name="toolTip">
          <string>TV Shows</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border: 0;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 0px
}</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../MediaElch.qrc">
           <normaloff>:/img/display_on_menu.png</normaloff>:/img/display_on_menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>52</width>
           <height>52</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonConcerts">
         <property name="toolTip">
          <string>Concerts</string>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
	border: 0;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 0px
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../MediaElch.qrc">
           <normaloff>:/img/concerts_menu.png</normaloff>:/img/concerts_menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>52</width>
           <height>52</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="movieSplitter">
          <property name="styleSheet">
           <string notr="true">QSplitter::handle {
	background-color: transparent;
}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="FilesWidget" name="filesWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
          </widget>
          <widget class="QScrollArea" name="scrollArea">
           <property name="styleSheet">
            <string notr="true">QScrollArea, #scrollAreaWidgetContents_2 {
	background-color: transparent;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>82</width>
              <height>735</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="MovieWidget" name="movieWidget" native="true"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="tvShowSplitter">
          <property name="styleSheet">
           <string notr="true">QSplitter::handle {
	background-color: transparent;
}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="TvShowFilesWidget" name="tvShowFilesWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
          </widget>
          <widget class="QScrollArea" name="scrollArea_2">
           <property name="styleSheet">
            <string notr="true">QScrollArea, #scrollAreaWidgetContents_3 {
	background-color: transparent;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>82</width>
              <height>735</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="TvShowWidget" name="tvShowWidget" native="true"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="SetsWidget" name="setsWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="concerts">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="concertSplitter">
          <property name="styleSheet">
           <string notr="true">QSplitter::handle {
	background-color: transparent;
}</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="ConcertFilesWidget" name="concertFilesWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
          </widget>
          <widget class="QScrollArea" name="scrollArea_3">
           <property name="styleSheet">
            <string notr="true">QScrollArea, #scrollAreaWidgetContents_4 {
	background-color: transparent;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>100</width>
              <height>30</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="ConcertWidget" name="concertWidget" native="true"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="GenreWidget" name="genreWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="CertificationWidget" name="certificationWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>34</width>
     <height>34</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MovieWidget</class>
   <extends>QWidget</extends>
   <header>movies/MovieWidget.h</header>
   <container>1</container>
   <slots>
    <signal>setActionSaveEnabled(bool,MainWidgets)</signal>
    <signal>setActionSearchEnabled(bool,MainWidgets)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>FilesWidget</class>
   <extends>QWidget</extends>
   <header>movies/FilesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TvShowFilesWidget</class>
   <extends>QWidget</extends>
   <header>tvShows/TvShowFilesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TvShowWidget</class>
   <extends>QWidget</extends>
   <header>tvShows/TvShowWidget.h</header>
   <container>1</container>
   <slots>
    <signal>sigSetActionSaveEnabled(bool,MainWidgets)</signal>
    <signal>sigSetActionSearchEnabled(bool,MainWidgets)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>SetsWidget</class>
   <extends>QWidget</extends>
   <header>sets/SetsWidget.h</header>
   <container>1</container>
   <slots>
    <signal>setActionSaveEnabled(bool,MainWidgets)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>ConcertFilesWidget</class>
   <extends>QWidget</extends>
   <header>concerts/ConcertFilesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConcertWidget</class>
   <extends>QWidget</extends>
   <header>concerts/ConcertWidget.h</header>
   <container>1</container>
   <slots>
    <signal>setActionSaveEnabled(bool,MainWidgets)</signal>
    <signal>setActionSearchEnabled(bool,MainWidgets)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>GenreWidget</class>
   <extends>QWidget</extends>
   <header>movies/GenreWidget.h</header>
   <container>1</container>
   <slots>
    <signal>setActionSaveEnabled(bool,MainWidgets)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>CertificationWidget</class>
   <extends>QWidget</extends>
   <header>movies/CertificationWidget.h</header>
   <container>1</container>
   <slots>
    <signal>setActionSaveEnabled(bool,MainWidgets)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../MediaElch.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonGenres</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMenuGenres()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCertifications</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMenuCertifications()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonMovies</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMenuMovies()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonMovieSets</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMenuMovieSets()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonTvshows</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMenuTvShows()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonConcerts</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onMenuConcerts()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>movieWidget</sender>
   <signal>setActionSaveEnabled(bool,MainWidgets)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSetSaveEnabled(bool,MainWidgets)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>movieWidget</sender>
   <signal>setActionSearchEnabled(bool,MainWidgets)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSetSearchEnabled(bool,MainWidgets)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>concertWidget</sender>
   <signal>setActionSaveEnabled(bool,MainWidgets)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSetSaveEnabled(bool,MainWidgets)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>concertWidget</sender>
   <signal>setActionSearchEnabled(bool,MainWidgets)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSetSearchEnabled(bool,MainWidgets)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tvShowWidget</sender>
   <signal>sigSetActionSaveEnabled(bool,MainWidgets)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSetSaveEnabled(bool,MainWidgets)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tvShowWidget</sender>
   <signal>sigSetActionSearchEnabled(bool,MainWidgets)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSetSearchEnabled(bool,MainWidgets)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>811</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setsWidget</sender>
   <signal>setActionSaveEnabled(bool,MainWidgets)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSetSaveEnabled(bool,MainWidgets)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>genreWidget</sender>
   <signal>setActionSaveEnabled(bool,MainWidgets)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSetSaveEnabled(bool,MainWidgets)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>certificationWidget</sender>
   <signal>setActionSaveEnabled(bool,MainWidgets)</signal>
   <receiver>MainWindow</receiver>
   <slot>onSetSaveEnabled(bool,MainWidgets)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onMenuGenres()</slot>
  <slot>onMenuCertifications()</slot>
  <slot>onMenuMovies()</slot>
  <slot>onMenuMovieSets()</slot>
  <slot>onMenuTvShows()</slot>
  <slot>onMenuConcerts()</slot>
  <slot>onSetSaveEnabled(bool,MainWidgets)</slot>
  <slot>onSetSearchEnabled(bool,MainWidgets)</slot>
 </slots>
</ui>
